### PR TITLE
Compatibility with xarray v0.16

### PR DESCRIPTION
**Pull request by malmans2**
* The stable release of xmitgcm is not compatible with xarray v0.16, but the master branch yes. Use `pip install git+` until next stable release
* `ax` and `subplot_kwargs` are now mutually exclusive in `xarray.plot()`, so I had to update a couple of things in the `horizontal_section` function
* I've added @Mikejmnez to the CODEOWNER list

### CODE REVIEW

**Comment by malmans2** [comment]
The codecov content has been erased. When we merge this, hopefully #147 will be resolved. I don't think @Mikejmnez can review yet (he'll be able to review as soon as we merge), so I need the approval from someone else.

**Review by asiddi24 — APPROVED** [review]
So,  I had to remove my previous envs and create an entirely new one to test this. Somehow, simply updating the old `ospy_tests` was having conflict issues. Anyways, it worked out. 

The horizontal section plotting seems to be smooth. I tested with the get_started dataset and it works fine.

Tests also passing on my local machine with 99% coverage. 

Looks like we can merge. @Mikejmnez welcome to the CODEOWNER list !

**Comment by malmans2** [comment]
Thanks!

**Pull request merged**
